playbooks/kubevirt.yml

```diff
@@ -0,0 +1,253 @@
+---
+- hosts: all
+  become: true
+  vars:
+    selinux_state: permissive
+    pod_network: 10.244.0.0/16
+    flannel_version: a70459be0084506e4ec919aa1c114638878db11b
+    multus_version: master
+    kubevirt_version: v0.13.2
+    ovs_cni_version: master
+  roles:
+    - selinux
+    - epel_repositories
+  tasks:
+    - name: disable swap
+      command: swapoff -a
+
+    - name: deconfigure swap
+      mount:
+        src: /swapfile
+        fstype: swap
+        path: none
+        state: absent
+    - name: modprobe br_netfilter
+      modprobe:
+        name: br_netfilter
+
+    - name: configure bridge iptables
+      sysctl:
+        name: "{{ item }}"
+        value: 1
+        sysctl_file: /etc/sysctl.d/k8s.conf
+      with_items:
+        - net.bridge.bridge-nf-call-ip6tables
+        - net.bridge.bridge-nf-call-iptables
+
+    - name: install docker
+      yum:
+        name: docker
+
+    - name: enable docker
```

Comment:
If installing Kubernetes why do you need docker?

Reply:
because I use docker as the runtime, was too lazy to use anything else and that worked

Reply:
Ahh Kubernetes doesn't come installing cri-o by default? I know it is technially the default now.

Reply:
I've been following https://kubernetes.io/docs/setup/independent/install-kubeadm/ and that said: " The container runtime used by default is Docker, which is enabled through the built-in dockershim CRI implementation inside of the kubelet."

Reply:
Fair enough if cri-o is too complicated to get setup right now.